### ISSUE TITLE
LPG Refill at Birkenhead Point

### ISSUE THREAD

**Issue by smr547** [CanberraOceanRacingClub/namadgi3#768]
The marina does NOT offer an LPG bottle refill service. However a refill service is available on the south side of the Iron Cove Bridge at

Metro Petroleum Rozelle
127 Victoria Rd, Rozelle NSW 2039

[See on Google Maps](https://maps.app.goo.gl/sPRHUfZmeeUgSYy58)

**Comment by pcbaston** [comment]
25Jan24 - Tried Metro Petroleum at Roselle but had no luck. 
Found the nearest place to be Metro Petroleum Leichhardt. Full bottle refill cost $19.    Fuel was very cheap there also.
Other yachties in the marina was happy when I advised them where bottles could be refilled. All petrol stations nearby only do Swap&Go.

Metro Petroleum Leichhardt
127-129 Marion St, Leichhardt NSW 2040

[Google Maps Directions](https://www.google.com/maps/dir/Birkenhead+Point+Marina,+Roseby+Street,+Drummoyne+NSW/Metro+Petroleum+Leichhardt,+127-129+Marion+St,+Leichhardt+NSW+2040/@-33.8695039,151.1380308,14z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x6b12af95253a0bf1:0x94cf11537273000a!2m2!1d151.1636379!2d-33.8564903!1m5!1m1!1s0x6b12b0055c60e4e9:0xfd328e76f058f9e1!2m2!1d151.1507412!2d-33.883907!3e0?entry=ttu)
